dartagnan/src/main/java/com/dat3m/dartagnan/verification/solving/RefinementSolver.java

```diff
@@ -498,7 +517,7 @@ private static CharSequence generateSummary(RefinementTrace trace, long boundChe
         StringBuilder message = new StringBuilder().append("Summary").append("\n")
                 .append(" ======== Summary ========").append("\n")
                 .append("Number of iterations: ").append(trace.iterations.size()).append("\n")
-                .append("Total native solving time(ms): ").append(totalNativeSolvingTime + boundCheckTime).append("\n")
+                .append("Total native solving time(ms): ").append(totalNativeSolvingTime).append("\n")
                 .append("   -- Bound check time(ms): ").append(boundCheckTime).append("\n")
```

Comment:
The indentation in the bound check time suggests "out of the total native, this was bound". But this is not the case anymore. So I would either keep the `boundCheckTime` for the total time, or change the indentation

Reply:
No, it is still part of the total time. By adding it explicitly we accidentally counted it twice before!
This is actually a bug fix rather than a change.